Userland/Libraries/LibTest/TestSuite.cpp

```diff
@@ -49,6 +50,12 @@ void add_test_case_to_suite(const NonnullRefPtr<TestCase>& test_case)
     TestSuite::the().add_case(test_case);
 }
 
+// Declared in TestCase.h
+void add_setup_to_suite(Function<void(void)>&& setup)
```

Comment:
the name `add_setup_to_suite` suggests to me that you can do it multiple times and they'll all be run? So i'd suggest either changing the name to `set_suite_setup_function` or similar, or making the setup thing a Vector<Function> if we want to support that use case.

Also few nitpicks here:
-  it's a bit more idiomatic to take sink arguments by value, and then move. Suggest removing && from function argument.

- I'd rather add a `set_suite_setup()` API to the test suite rather than assigning to a public member.